install_config/aggregate_logging.adoc

```diff
@@ -528,12 +525,19 @@ For example, the selector could be `node-type=infrastructure`. At least
 one active node must have this label before Elasticsearch will deploy.
 
 |`openshift_logging_elasticsearch_kibana_index_mode`
-|The default value, `unique`, allows users to each have their own Kibana index. In
+a|The default value, `unique`, allows users to each have their own Kibana index. In
 this mode, their saved queries, visualizations, and dashboards are not shared.
 
 You may also set the value `shared_ops`. In this mode, all operations users
 share a Kibana index which allows each operations user to see the same
-queries, visualizations, and dashboards.
+queries, visualizations, and dashboards. To determine if you are an operations user:
+
+----
+#oc auth can-i view pod/logs -n default
+yes
+----
+
+If you do not have proper access, contact your cluster administrator.
```

Comment:
s/If you do not have proper access/If you cannot view the pod logs,
IBM Style says:
```
proper: Do not use to mean “correct,” “applicable,” or “appropriate.”
```